solutions/USACO dec17-the-bovine-shuffle-silver.mdx

```diff
@@ -0,0 +1,84 @@
+---
+id: usaco-764
+source: USACO Silver 2017 December
+title: The Bovine Shuffle
+author: Jesse Choe
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_shuffle_silver_dec17.html)
+
+Time Complexity: $\mathcal{O}(N)$
```

Comment:
time complexity should be bolded (as it is in the example)

Reply:
Isn't it already bolded? I just copied and pasted that from the example.

Reply:
No.
<img width="713" alt="Screen Shot 2021-02-10 at 10 15 27 AM" src="https://user-images.githubusercontent.com/21228024/107529492-ec5a5780-6b88-11eb-8e5c-5f21d002001d.png">